### PR TITLE
feat(No-id): Wire up files context and navigation

### PR DESCRIPTION
**Pull request by knguyenrise8**
# Pull Request
Aug-13-2024 15-42-12

## Description
 - wires up navigation to route `/new-template/annotate`
 - creates Files Context
 - updates unit tests

## Related Issues
[Link any related issues or tasks from your project management system.]

## Additional Notes
[Add any additional context or notes that reviewers should know about.]

## Checklist
Please review and complete the following checklist before submitting your pull request:

- [ ] I have ensured that the pull request is of a manageable size, allowing it to be reviewed within a single session.
- [ ] I have reviewed my changes to ensure they are clear, concise, and well-documented.
- [ ] I have updated the documentation, if applicable.
- [ ] I have added or updated test cases to cover my changes, if applicable.
- [ ] I have minimized the number of reviewers to include only those essential for the review.
- [ ] I have notified teammates in the review thread to build awareness.

## Checklist for Reviewers
Please review and complete the following checklist during the review process:

- [ ] The code follows best practices and conventions.
- [ ] The changes implement the desired functionality or fix the reported issue.
- [ ] The tests cover the new changes and pass successfully.
- [ ] Any potential edge cases or error scenarios have been considered.

### CODE REVIEW

File: OCR/frontend/package.json [hunk 1]
```diff
@@ -8,14 +8,16 @@
     "build": "tsc -b && vite build",
     "lint": "eslint .",
     "preview": "vite preview",
-    "test": "vitest",
+    "test": "vitest run --coverage.all",
```
Comment (zdeveloper) [inline OCR/frontend/package.json:11]: I would recommend separating this to a different command, we can do that in a future PR

File: OCR/frontend/src/componets/FileInput.tsx [hunk 2]
```diff
@@ -0,0 +1,246 @@
+import React, {
```
Comment (knguyenrise8) [inline OCR/frontend/src/componets/FileInput.tsx:1]: had to copy pasta these two components from USDWS bc it kept erroring out when importing

**Review by zdeveloper — COMMENTED** [review]
LGTM, Thank you!

**Review by zdeveloper — APPROVED** [review]
